rockcraft.yaml

```diff
@@ -0,0 +1,146 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+---
+# https://github.com/apache/superset/blob/master/Dockerfile
+name: charmed-superset-rock
+base: ubuntu@22.04
+version: 2.1.0-22.04-edge
+summary: Charmed Superset ROCK OCI
+description: |
+  Superset is a business intelligence tool that is fast, lightweight,
+  intuitive, and loaded with options that make it easy for users
+  to explore and visualize their data.
+license: Apache-2.0
+
+services:
+  superset-ui:
+    override: replace
+    summary: "superset-ui service"
+    startup: disabled
+    command: "/app/k8s/k8s-bootstrap.sh"
+    environment:
+      CHARM_FUNCTION: "app-gunicorn"
+      SUPERSET_SECRET_KEY: "supersetR0cks!"
+      ADMIN_PASSWORD: "admin"
+      SUPERSET_LOAD_EXAMPLES: True
```

Comment:
Note that the  `on-success` and `on-failure` actions are `restart` by default. [See Pebble readme](https://github.com/canonical/pebble?tab=readme-ov-file#layer-specification).

You might want to make that explicit in the rockcraft.yaml to avoid any surprises.